src/twisted/trial/newsfragments/9962.misc

```diff
@@ -0,0 +1 @@
+remove _twisted.trial.util:runSequentially
```

Comment:
The follow the style guideline, this would be "twisted.trial.util._runSequentially has been removed."

However, it's a private API so it doesn't seem worth text in the news file anyway.   Also worthy of note may be that .misc files have their contents discarded anyway so there's no reason to put any text here anyway.

Reply:
> Please note that the trac ticket, news fragment, and review submission portions of this process apply to all pull requests, no matter how small; if you don't do them, it's likely that nobody will even notice your PR needs a review.

is there a spec of when the trac ticket/news fragment is needed?

Reply:
> is there a spec of when the trac ticket/news fragment is needed?

Yes, it's linked from the checklist template.